tools/shared.py

```diff
@@ -669,7 +653,11 @@ def get_clang_native_env():
     return CACHED_CLANG_NATIVE_ENV
   env = os.environ.copy()
 
-  if WINDOWS:
+  if MACOS:
+    path = run_process(['xcrun', '--show-sdk-path'], stdout=PIPE).stdout.strip()
+    logging.debug('Using MacOS SDKROOT: ' + path)
+    env['SDKROOT'] = path
```

Comment:
Is this a macOS specific thing?

Reply:
Yes, `$SDKROOT` is used by clang and XCode tools to find the MacOS/iOS SDKs on the system, and the `xcrun` and `xcode-select` tools are command-line utilities for managing them.